### PR TITLE
Fix HA integration tests

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

Fix integration tests that are failed because of FLAdminAPI related changes.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/fuel/utils/time_utils.py [hunk 1]
```diff
@@ -28,3 +30,33 @@ def time_to_string(t) -> str:
         return "N/A"
 
     return time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(t))
+
+
+def collect_time(func):
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/fuel/utils/time_utils.py:35]: sorry, I will get rid of changes that are unrelated to this PR

**Review by yhwen — COMMENTED** [review]
The HA error is caused by missing the authentication check for the client pull_task and submit_result requests during the CellNet integration migration. I am doing the changes to add this authentication function back. That would be the consistent approach with the 2.2. Please undo this change.

**Review by nvkevlu — APPROVED** [review]
LGTM, thanks

**Pull request merged**